frontend/src/features/admin-form/settings/SettingsGeneralPage.tsx

```diff
@@ -21,6 +31,14 @@ export const SettingsGeneralPage = (): JSX.Element => {
       <FormIssueNotificationToggle />
       <Divider my="2.5rem" />
       <FormDetailsSection />
+      {isTest ||
```

Comment:
just a note: this short-circuiting `isTest` evaluation could mean:
- when in Storybook (`env.STORYBOOK_NODE_ENV === 'test'`)
- the string 'true' might appear on the SettingsGeneralPage